### PR TITLE
Pol-103 - Monthly Actual v. Budgeted Spend Report

### PR DESCRIPTION
**Pull request by rlafleur**
### Description

[Describe what this change achieves]

### Issues Resolved

[List any existing issues this PR resolves]

### Contribution Check List

- [x] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: cost/budget_v_actual/monthly_budget_v_actual.pt [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+
+|Range|Current|Budget|Overbudget|
+|:---:|------:|-----:|---------:|
+{{ range data.reportData -}}
+  | {{ .range }} | {{ .current }} | {{ .budget }} | {{ .overbudget }} |
+{{ end -}}  
+  
+### For more detailed cost information, visit [Optima](https://analytics.rightscale.com/orgs/{{ rs_org_id }}/dashboard).
+  
+For more information on this report, please view the [README](https://github.com/rightscale/policy_templates/tree/master/cost/scheduled_reports).
```
Comment (cdwilhelm) [inline cost/budget_v_actual/monthly_budget_v_actual.pt]: This link is wrong, it should point to the expected path once released.

File: cost/budget_v_actual/monthly_budget_v_actual.pt [hunk 2]
```diff
@@ -0,0 +1,399 @@
+name "Monthly Actual v. Budgeted Spend Report"
+rs_pt_ver 20180301
+type "policy"
+short_description "This policy allows you to set up scheduled reports that will provide monthly actual v. budgted cloud cost across all resources in the Billing Center(s) you specify, delivered to any email addresses you specify."
```
Comment (cdwilhelm) [inline cost/budget_v_actual/monthly_budget_v_actual.pt]: Needs link to README and docs.rightscale.com
budgeted is misspelled.

File: cost/budget_v_actual/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# Monthly Actual v. Budgeted Spend Report
+
+**As a best practice, this policy should only be applied to the Master Account, and not to each individual RightScale Account. The monthly budget inputs are compared against the current month's actual cloud costs of the applied policy. Reapply this policy on an annual basis to reflect the current year's monthly budgeted cloud costs.**
```
Comment (cdwilhelm) [inline cost/budget_v_actual/README.md]: Please follow the README_GUIDELINE for consistency across all policy README.

Comment (cdwilhelm) [inline cost/budget_v_actual/README.md]: This sentence isn't needed.  As a best practice, this policy should only be applied to the Master Account, and not to each individual RightScale Account.
Instead add the new tenancy field.  tenancy "single"
Move the rest of the bolded detail into Functional Details

**Comment by rlafleur** [comment]
@cdwilhelm ready for review.

**Pull request merged**